src/TodoList.tsx

```diff
@@ -0,0 +1,40 @@
+// [TO IMPLEMENT] 
+// @Ethan I do understand that eventually it is decided for me how to write company's code.
+// But for the sake of learning, when is TS most commonly used?
+// Do you mind create a standard TS working pattern with me. Thanks
+
+interface Job {
+    title: string;
+    id: number;
+  }
+
+const list:Job[] = [
+    {
+        title: 'Cleaning',
+        id: 0,
+    }, {
+        title: 'Groceries',
+        id: 1,
+    }, {
+        title: 'Work',
+        id: 2,
+    },
+    {
+        title: 'Sleep',
+        id: 3,
+    },
+];
+
+const TodoList = () => (
+    <ul className="list-wrapper">
+        {list.map((item:Job) => (
+            <li key={item.id}>
+            <input type="checkbox" id={item.title} name={item.title} value={item.title}/>
```

Comment:
Your return on this map is missing a `key` property on its parent element. Since there are two elements, what should you do?

Reply:
Ah yes, it was not indented properly. It's good practice to run a formatter to avoid those kind of issues. I think prettier is the standard for js/ts files.